content/blog/overview-of-the-platform-level-data-model-for-redfish®-device-enablement-standard.md

```diff
@@ -0,0 +1,154 @@
+            {
+                "@odata.id": "/redfish/v1/Systems/1/Storage/DE07C000/Drives/1"
+            }
+        ]
+    }
+}
+```
+
+The request will be transferred to the device and analyzed. If the device can process it, you will receive a `201 Created` HTTP response code. The just created Logical Drive can be used immediately by the operating system without any system reboot. You may have to ask the OS to re-scan storage devices to discover this newly created LD.
+
+The deletion of a RDE Logical Drive is straight forward; you just have to send a DELETE request toward the LD URI:
+
+```Bash
+curl --request DELETE "$iloURI/redfish/v1/Systems/1/Storage/DE07C000/Volumes/1"
+```
+
+**NOTE**: For backward compatibility concerning HPE Smart Array SR devices, HPE Gen10 and Gen10 Plus servers offer the possibility to manage those devices using both `SmartStorageConfig` and `Storage` data types. However, always stick to a single management data type. Mixing those data types may lead to hazardous results.
+
+### Network adapters
+
+Several network adapter suppliers implement PLDM for RDE in their devices. The [Intel E810 XXVDA2 Network Adapter](https://www.hpe.com/psnow/doc/a00073559enw.html?jumpid=in_pdp-psnow-qs) support this new technology in its firmware version 3.10. This firmware is packaged in a file called `HPE_E810_XXVDA2_SD_3p10_PLDMoMCTP_8000AD4A.fwpkg` mentioning explicitly PLDM over MCTP, meaning that PLDM messages between the Management Controller and the device are transported by the Management Component Transport Protocol mentioned above.
+
+When this firmware is deployed, a network adapter containing string `/DE` is present under the `NetworkAdapter.` data type location. The next picture extracts three properties (URI, Name and Settings URI) from the `NetworkAdapter.` data type of a system containing two network adapters: an HPE 631FLR-SFP28 and an Intel E810. The HPE 631FLR-SFP28 adapter does not support RDE and is referred to a `/DC` device. Moreover, it contains a settings URI used to store modifications before being transferred to the device during the next system reboot.
+
+The Intel E810 adapter is referred to a `/DE` device supporting RDE. Moreover, it does not contain any settings URI because modifications are transferred to the device in real time by the iLO and no server reboot is required to take those modification into account.
+
+![Setting an RDE network adapter property with iLOrest](/img/networkadapters.png "Setting an RDE network adapter property with iLOrest")
+
+The following picture shows a real-time property modification on port 0 of an Intel E810 network adapter, using [iLOrest](http://hpe.com/info/resttool) version 3.5.
+
+The first instruction selects explicitly the `port.v1_3_1` data type that includes this adapter. Using a `ilorest select port.` data would also select other data versions (i.e. data types related to non RDE adapters). If several data types are selected, the filtering of properties is harder.
```

Comment:
swap placement of "selects" and "explicitly" so it reads:
"The first instruction explicitly selects the..."